Simplified/NYPLBook.h

```diff
@@ -19,6 +14,7 @@ typedef NS_ENUM(NSInteger, NYPLBookAvailabilityStatus) {
 
 @property (nonatomic, readonly) NYPLBookAcquisition *acquisition;
 @property (nonatomic, readonly) NSString *authors;
+@property (nonatomic, readonly) NSArray *authorLinks;
```

Comment:
Would it be possible to use generics here? I know the class doesn't elsewhere yet but there should be no harm in starting.